fk/keycreate.go

```diff
@@ -248,11 +250,18 @@ func verifyEmailMatchesKeyInAPI(
 
 	retrievedKey, err := pgpkey.LoadFromArmoredPublicKey(armoredKey)
 	if err != nil {
-		return false, fmt.Errorf("Failed to load armored key: %v", err)
+		return false, fmt.Errorf("failed to load armored key: %v", err)
 	}
 	if retrievedKey.Fingerprint() == fingerprint {
 		return true, nil
 	}
+	retrievedEmail, err := retrievedKey.Email()
```

Comment:
This *new* test is cute: ensure the API didn't return as a duff key which actually has no email addresses on it.

But did you mean to introduce it? Are there other consequences?